app/services/error_details/create_service.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module ErrorDetails
+  class CreateService < BaseService
+    def initialize(params:, owner:, organization:, integration: nil)
+      super(params:, integration:, owner:, organization:)
+    end
+
+    def call
+      result = super
+      return result if result.error
```

Comment:
I think in general we use `result.success?` instead of looking at the error. Especially because you're model here is an error, it gets extra confusing 😁